src/spikeinterface/core/basesorting.py

```diff
@@ -263,7 +263,7 @@ def get_unit_property(self, unit_id, key):
 
     def get_total_num_spikes(self):
         warnings.warn(
-            "Sorting.get_total_num_spikes() is deprecated, se sorting.count_num_spikes_per_unit()",
+            "Sorting.get_total_num_spikes() is deprecated and will be removed in spikeinterface 0.102, se sorting.count_num_spikes_per_unit()",
```

Comment:
```suggestion
            "Sorting.get_total_num_spikes() is deprecated and will be removed in spikeinterface 0.102, use sorting.count_num_spikes_per_unit()",
```

Reply:
Just because se might make people think of sorting extractors instead of use. I know this will be removed, but just to be clear.